griptape/drivers/image_query_model/base_image_query_model_driver.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
+from attr import define
+from griptape.artifacts import TextArtifact, ImageArtifact
+from griptape.mixins import SerializableMixin
+
+
+@define
+class BaseImageQueryModelDriver(SerializableMixin, ABC):
+    @abstractmethod
+    def construct_image_query_request_parameters(self, query: str, images: list[ImageArtifact]) -> dict:
```

Comment:
For consistency with `BaseImageGenerationModelDriver`, I think we should shorten to `image_query_request_parameters`